WgServerforWindows/Models/ClientConfiguration.cs

```diff
@@ -50,21 +50,49 @@ public ClientConfiguration(ClientConfigurationList parentList)
                 DependencySatisfiedFunc = prop => string.IsNullOrEmpty(prop.Validation?.Validate?.Invoke(prop)),
                 Action = (conf, prop) =>
                 {
-                    IPNetwork serverNetwork = IPNetwork.Parse(serverConfiguration.AddressProperty.Value);
-                    var possibleAddresses = serverNetwork.ListIPAddress().Skip(2).SkipLast(1); // Skip reserved .0 and .1 and .255.
+                    WaitCursor.SetOverrideCursor(Cursors.Wait);
+
+                    var serverAddresses = serverConfiguration.AddressProperty.Value
+                        .Split(new[] { ',' })
+                        .Select(a => IPNetwork.Parse(a.Trim()))
+                        .ToList(); // Prevent multiple enumeration
+
+                    var currentClientAddresses = prop.Value
+                        .Split(new[] { ',' })

```

Comment:
```suggestion
                        .Split(',')
```
I tested this version and it works fine. Also, doesn't `new[] {}` makes memory allocation compared to this version?

Reply:
Good catch! I fixed it later in the same file as well.